src/BinFile/PE.fs

```diff
@@ -103,7 +104,16 @@ type PEFileInfo (bytes, path, ?rawpdb) =
     if onlyDef then getExportSymbols pe else getAllDynamicSymbols pe
     |> List.toSeq
 
-  override __.GetRelocationSymbols () = Utils.futureFeature ()
+  override __.GetRelocationSymbols () =
+    pe.RelocBlocks
+    |> Seq.collect(fun block -> block.Entries |> Seq.map(fun entry -> (block, entry)))
```

Comment:
Place a space after `Seq.collect`, and maybe put `block.Entries ...` to the next line for better readability.